tests/mfx_dispatch/linux/mfx_dispatch_test_cases_plugins.cpp

```diff
@@ -80,6 +95,140 @@ TEST_P(DispatcherPluginsTestParametrized, ShouldLoadFromGoodPluginsCfgCorrectly)
     ASSERT_EQ(sts, MFX_ERR_NONE);
 }
 
-INSTANTIATE_TEST_CASE_P(EnumeratingPlugins, DispatcherPluginsTestParametrized, ::testing::Range((size_t) 0, TEST_PLUGIN_COUNT));
+INSTANTIATE_TEST_CASE_P(EnumeratingPlugins, DispatcherPluginsTestParametrized, ::testing::Range((size_t)0, TEST_PLUGIN_COUNT));
+
+TEST_F(DispatcherPluginsTestParametrized, ShouldFailIfPluginHasNoSymbols)
```

Comment:
The test fixture used here and in other similar spots should be `DispatcherPluginsTest`, not `DispatcherPluginsTestParametrized`.

Reply:
I agree with you. I will fix it.